scripts/apps/authoring/views/authoring-topbar.html

```diff
@@ -86,7 +86,7 @@
     </div>
     <div class="subnav__button-stack subnav__button-stack--default">
       <button id="Edit" class="btn btn--medium btn--primary" ng-if="item._type !== 'legal_archive'"
-        ng-show="action === 'view' && !item._editable && itemActions.edit"
+        ng-show="action === 'view' && !item._editable && (itemActions.edit || item.state === 'unpublished')"
```

Comment:
I think we should rather set `itemActions.edit` when state is unpublished rather than dealing with that logic here in the template

Reply:
@petrjasek setting `itemActions.edit` to true will not show the edit button, but it would rather open the article in edit mode directly. Will that be ok?

Reply:
not sure about that. @karelpetrak ?

Reply:
@GyanP @petrjasek Does it mean that whenever a user doubleclicks an unpublished story they get directly to the edit mode? Even if the do it on the story's "instance" which has "published" label?

Reply:
@petrjasek Karel, have agreed to how it behaves now.